m3u8/parser.py

```diff
@@ -412,8 +412,7 @@ def _cueout_envivio(line, state):
     res = re.match('.*DURATION=(.*),.*,CUE="(.*)"', value)
     if res:
         return (res.group(2), res.group(1))
-    else:
-        return None
+    return None
```

Comment:
`None` will be returned implicitly, so this statement is only useful if it's necessary to be explicit for some reason

Reply:
I think this case is one of those "explicit is better than implicit"